app.js

```diff
@@ -724,4 +724,49 @@ document.addEventListener('DOMContentLoaded', () => {
   email.addEventListener('input', () => {
     emailError.style.display = 'none';
   });
+
+  // add local storage for forma data
+
+  // initialising form content
+  const formSt = document.querySelector('#form');
+  const inputName = document.querySelector('#name');
+  const inputEmail = document.querySelector('#email');
+  const inputTextArea = document.querySelector('#comment_text');
+
+  // get stored data values from the local storage if they exist
+  const savedData = JSON.parse(localStorage.getItem('formData')) || {};
+
+  // fill in the inputs
+  if (savedData.inputName) {
+    inputName.value = savedData.inputName;
+  }
+  if (savedData.inputEmail) {
+    inputEmail.value = savedData.inputEmail;
+  }
+  if (savedData.inputTextArea) {
+    inputTextArea.value = savedData.inputTextArea;
+  }
+
+  // add an event listener to update the input fields if they change
+  inputName.addEventListener('input', (event) => {
+    savedData.inputName = event.target.value;
+    localStorage.setItem('formData', JSON.stringify(savedData));
+  });
+
+  inputEmail.addEventListener('input', (event) => {
+    savedData.inputEmail = event.target.value;
+    localStorage.setItem('formData', JSON.stringify(savedData));
+  });
+
+  inputTextArea.addEventListener('input', (event) => {
+    savedData.inputTextArea = event.target.value;
+    localStorage.setItem('formData', JSON.stringify(savedData));
+  });
+
```

Comment:
Currently, this text area field is not updating the local storage when data is being added to it. Please consider resolving it to meet the project requirements. 


```
{
    "inputName": "ABDUL ALI",
    "inputEmail": "example@gmail.com"
}
```

**[OPTIONAL]** The project submits without even adding anything in the text area field consider making it required to make it more professional.

Reply:
### **As per my previous CR 🔍  i was requested ♻️  to make some changes like :**
**-Add the email 📫  input to the local storage 🥇.**

as per my last commit 🟢  to the pull request i have fixed  🧑‍🔧  all the issues above. 🏅  
I hope to hear from you my CR 🤺  soon!.